app/views/feeds/_form_megaphone_config.html.erb

```diff
@@ -30,6 +37,13 @@
             <%= field_help_text t(".help.network_name") %>
           </div>
         </div>
+        <div class="col-12 mb-4">
+          <div class="form-floating input-group">
+            <%= config_form.select :advertising_tags, feed.advertising_tag_options, {include_blank: false}, multiple: true %>
```

Comment:
Are these quick "enough" that we don't need to worry about caching them?

(Seems like this will hit megaphone on every page load... but it's so rare that someone would (A) have a megaphone feed, and (B) be on this page editing things).

Reply:
so far it's been very quick when I test it, so I didn't worry about it.